src/test/resources/split_test.vcf

```diff
@@ -31,4 +31,4 @@
 22	1180	.	CAGGCG	C	.	.	.	GT:AD:DP:GQ:PL	0/0:18,0:18:45:0,45,675	0/0:9,0:9:21:0,21,315	0/0:18,0:18:42:0,42,630	1/1:1,23:24:36:814,36,0	0/0:52,0:52:99:0,120,1800	./.	0/1:38,29:67:99:872,0,1192	0/1:38,29:67:99:872,0,1192
 22	1183	.	G	C,*	.	.	.	GT:AD:DP:GQ:PL	0/0:4,0,0:4:12:0,12,136,15,134,153	0/0:4,0,0:4:12:0,12,136,15,134,153	0/1:2,1,0:3:13:13,0,42,22,44,73	2/2:0,0,6:6:18:150,168,233,18,21,0	1/1:0,1,0:1:2:8,2,0,11,3,31	./.	0/2:5,0,4:9:70:70,96,290,0,110,110	1/2:0,2,2:4:16:115,28,16,64,0,59
 22	1783	.	TA	TAA,T	.	.	.	GT:AD:DP:GQ:PL	0/0:4,0,0:4:12:0,12,136,15,134,153	0/1:2,1,0:3:13:13,0,42,22,44,73	1/1:0,1,0:1:2:8,2,0,11,3,31	0/2:5,0,4:9:70:70,96,290,0,110,110	1/2:0,2,2:4:16:115,28,16,64,0,59	2/2:0,0,6:6:18:150,168,233,18,21,0	./.	./.
-22	2167	.	AAAAC	AAAACAAACAAAC,AAAACAAAC,A	.	.	.	GT:AD:DP:GQ:PL	0/0:5,0,0,0:5:16:0,17,287,16,152,136,17,147,134,150	0/1:1,1,0,0:2:9:33,0,19,10,16,21,19,9,18,22	1/1:0,1,0,0:1:2:81,7,0,31,2,23,91,9,33,177	0/2:5,0,0,0:5:1:37,1,262,0,127,111,64,131,118,298	1/2:0,1,1,0:2:22:85,25,22,32,0,24,97,32,34,234	0/3:0,0,0,0:0:0:1,1,2,1,2,1,0,1,1,0	1/3:2,1,1,0:4:1:47,1,31,3,11,17,15,0,10,15	3/3:0,0,0,1:1:3:20,20,20,14,14,11,3,3,3,0
+22	2167	.	AAAAC	AAAACAAACAAAC,AAAACAAAC,ACAAC	.	.	.	GT:AD:DP:GQ:PL	0/0:5,0,0,0:5:16:0,17,287,16,152,136,17,147,134,150	0/1:1,1,0,0:2:9:33,0,19,10,16,21,19,9,18,22	1/1:0,1,0,0:1:2:81,7,0,31,2,23,91,9,33,177	0/2:5,0,0,0:5:1:37,1,262,0,127,111,64,131,118,298	1/2:0,1,1,0:2:22:85,25,22,32,0,24,97,32,34,234	0/3:0,0,0,0:0:0:1,1,2,1,2,1,0,1,1,0	1/3:2,1,1,0:4:1:47,1,31,3,11,17,15,0,10,15	3/3:0,0,0,1:1:3:20,20,20,14,14,11,3,3,3,0
```

Comment:
I don't know enough about splitting -- can you write a description why this is correct or provide a link? The output below is not what I would expect. Also, is this just to prevent future errors? I see nothing else is modified.

Reply:
split_multi calls minrep on all the split variants, and moves if necessary. (there's a flag for the option to throw an error, instead.) I think minrep removes identical sequences from the end, and then from the start, adjusting the locus position if necessary?

yeah, to prevent future errors. Currently we have nothing that is testing the case in splitmulti where the split variants aren't already sorted/don't need to be moved.

Reply:
I'm not sure about its validity w.r.t the VCF spec, but it's something that we support when splitting so I feel like it should be a valid test case regardless. I think it's conceptually the same as the second example in the `min_rep` docs:

https://hail.is/docs/devel/functions/genetics.html#hail.expr.functions.min_rep

and I could make the alt something like `AAAAA` if you'd prefer.